### DISCUSSION TITLE
How to use YimMenu’s Model Swapper?

### DISCUSSION THREAD

**Discussion by KnollenRick** [YimMenu/YimMenu#1388] [Q&A]
Hi, I have a quick question: How does the Model Swapper works? I have copied the entity’s hash using the Context Menu, and I pasted it into the Model Swapper, but nothing happened. I want to turn myself into a NPC to troll a friend. XD

Thank U

**Comment by Yimura** [comment]
Model swapper is a local feature only.

If you want to change your model for them you need to modify your player model under spoofing settings:
![image](https://github.com/YimMenu/YimMenu/assets/24669514/00be38ef-d859-44b2-bd43-66b9267a812e)

**Reply by KnollenRick** [reply]
When I activate this feature, YimMenu is notifying me instantly and permanently, that EVERY player is trying to crash me... 😅

**Reply by KnollenRick** [reply]
And how can I change my player model to a cop for example?

**Reply by KnollenRick** [reply]
@Yimura
